### DISCUSSION TITLE
Why does returning fail in form action use HTTP status code 200?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [sveltejs/kit#11231] [Ideas]
I'm starting to implement a bunch of forms using Sveltekit and wonder, why do we return HTTP status code 200 with this code?

```
return fail(400, { form })
```

Would it not be more correct from an HTTP perspective to return status code 400)?

**Comment by teemingc** [comment]
Here is a previous discussion I've found on the matter https://github.com/sveltejs/kit/issues/7233#issuecomment-1279779888
